examples/conda/build.envd

```diff
@@ -1,29 +1,9 @@
 def build():
-    config.conda_channel(channel="""
```

Comment:
Maybe you can avoid changing the code here and create a new example conda-env-yaml for the new feature.

Reply:
ok